packages/api/src/schema/schema.yaml

```diff
@@ -1523,6 +1541,54 @@ components:
                           description:
                             Will be added to the pinata_secret_api_key header.
 
+    signing-key:
+      type: object
+      additionalProperties: false
+      required: [publicKey]
+      table: signing_key
+      properties:
+        id:
+          type: string
+          readOnly: true
+          index: true
+          example: 78df0075-b5f3-4683-a618-1086faca35dc
+        name:
+          type: string
+          description: Name of the signing key
+        createdAt:
+          type: number
+          readOnly: true
+          description:
+            Timestamp (in milliseconds) at which the signing-key was created
+          example: 1587667174725
+        lastUsedAt:
```

Comment:
This is not used yet right? Will be on the follow-up PR for the `/gating` or `/access-control` API or sth like that

Reply:
Yeah, it's just defined but not used yet!